cdap-ui/app/features/hydrator/bottompanel.less

```diff
@@ -249,6 +249,10 @@ body.theme-cdap.state-hydrator {
     .output-schema {
       .schema-propagation-confirm {
         padding-top: 10px;
+
+        .confirmation-button {
```

Comment:
@elfenheart does .confirmation button need to be nested inside .schema-propagation-confirm? If it will work without the additional specificity, we should move it directly underneath .schema-propagation-confirm